.ado/templates/build-rnw.yml

```diff
@@ -31,7 +31,8 @@ steps:
       installVsComponents: ${{ parameters.installVsComponents }}
       debug: ${{ parameters.debug }}
 
-  - task: NuGetCommand@2
+  # NuGetCommand@2 workaround: https://developercommunity.visualstudio.com/content/problem/288534/vsts-yaml-build-failure-the-task-name-nugetcommand.html
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
```

Comment:
i wonder if this is even required now that we can do `/restore /p:RestorePackagesConfig=true` ?

Reply:
The issue was brought up on #6958, but I'm not taking on a further refactor for the purposes of getting codesigning unblocked.